scripts/generate.ts

```diff
@@ -101,22 +102,26 @@ const parseTutorial = (directory: string): TutorialItem => {
         continue;
       }
 
-      embeddedCodeBlock =
-        '```' +
-        language +
-        '\n' +
-        lines.slice(startLine - 1, endLine).join('\n') +
-        '\n```';
-    } else {
-      embeddedCodeBlock = '```' + language + '\n' + codeContent + '\n```';
-    }
+      codeContent =  lines.slice(startLine - 1, endLine).join('\n')
+    } 
+
+    let files = [];
+
+    files.push({ path: codeFilePath, content: codeContent })
+
+    filePathDependsOn.forEach(function(filePath) {
+      const codeFilePath = path.join(directory, filePath);
+      const codeContent = fs.readFileSync(codeFilePath, 'utf-8');
+      files.push({ path: codeFilePath, content: codeContent })
+    })
+
+    const filesEncoded = encodeURIComponent(JSON.stringify(files)) 
+
+    const embeddedCodeBlock = `<Playground open="${codeFilePath}" files="${filesEncoded}">`;
```

Comment:
Is the prop `open` leftover? I don't see it used in the`Playground` component

Reply:
I would also add a sentence or two about the `depends_on_file` syntax that was added in the repo README

Reply:
* `open` will be used in a future version playground, let me pass it to the component. 
* docs added by 61c1541